examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>Why do you need time here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>